README.md

```diff
@@ -74,10 +75,10 @@ Procedure:
 
 ## Labels
 
-| Label Name | Purpose | How to use this label | Who should use this label |
-| ------ | ------ | ------ | ------ |
-| `sig/foo` | Denotes the SIG(s) which owns this enhancement e.g., `SIG Foo` | Set the label using the comment `/sig foo` (on a separate line) | Anyone |
-| `kind/feature` | Denotes that the issue should be tracked as an enhancement (All enhancement issues should be marked with this label) | Set the label using the comment `/kind feature` (on a separate line) | Anyone |
-| `tracked/yes` | Denotes an issue has been reviewed by a Feature Maintainer (SIG PM / SIG Release) and is actively tracked for the current milestone | Manually set | Feature Maintainers (SIG PM / SIG Release) ONLY |
-| `tracked/no` | Denotes an issue has been reviewed by a Feature Maintainer (SIG PM / SIG Release) and will not actively tracked for the current milestone | Manually set | Feature Maintainers (SIG PM / SIG Release) ONLY |
-| `stage/{alpha,beta,stable}` | Denotes the stage of an issue in the features process. | Set the label using the comment `/stage alpha` (on a separate line) | Anyone |
+| Label Name                  | Purpose                                                                                                                                   | How to use this label                                                | Who should use this label                       |
```

Comment:
Please drop the space additions. It suggests that we need to update the spacing every time we change the width of a column and that is automatically figured out when the document is rendered in markdown.

Reply:
Agreed. 

I added that as the document is also readable when opened in raw format.